admin/banner/class-admin-banner-sidebar.php

```diff
@@ -118,7 +118,7 @@ protected function get_services_spot() {
 		$service_spot->set_description(
 			sprintf(
 				/* translators: %1$s expands to anchor start text with link to the Yoast Services page, %2$s to Yoast, %3$s is the a-closing tag.  */
-				__( 'Don\'t want dive into SEO yourself? %1$sLet team %2$s help you!%3$s' ),
+				__( 'Don\'t want to dive into SEO yourself? %1$sLet team %2$s help you!%3$s' ),
```

Comment:
Since we've used before I would say the exclamation mark inside the link is fine. If we want to do it otherwise, we should check all strings for consistency.